src/components/NavUtility/NavUtility.tsx

```diff
@@ -5,11 +5,12 @@ import { NavItemUtility } from '../NavPrimary';
 export interface NavUtilityProps {
   utilityItems: NavItemUtility[];
   ariaLabel?: string;
+  className?: string;
 }
 
-const NavUtility = ({ utilityItems, ariaLabel = 'Utility' }: NavUtilityProps): JSX.Element => {
+const NavUtility = ({ utilityItems, ariaLabel = 'Utility', className }: NavUtilityProps): JSX.Element => {
   return (
-    <nav className="bsds-nav-utility" aria-label={ariaLabel}>
+    <nav className={`bsds-nav-utility ${className || ''}`} aria-label={ariaLabel}>
```

Comment:
sorry, the requirements were lacking here too. i lumped this in as part of excluding primary nav. can we revert this file?